CONTRIBUTING.md

```diff
@@ -557,7 +557,7 @@ Names of files and directories should be in lowercase, with dashes between words
 
   ```nix
   foo {
-    arg = ...;
+    arg = "...";
```

Comment:
```suggestion
    arg = <...>;
```

This looks more like meta-syntax, even though it also happens to be valid regular syntax.

Reply:
I had the same feedback. What's the best way to address it?